activitypub/activitypub.go

```diff
@@ -2,19 +2,18 @@ package activitypub
 
 import (
 	"context"
-	"fmt"
 	"log"
 	"net/http"
 	"net/url"
 
 	"github.com/exlibris-fed/exlibris/activitypub/clock"
 	"github.com/exlibris-fed/exlibris/activitypub/database"
-	"github.com/exlibris-fed/exlibris/model"
+	//"github.com/exlibris-fed/exlibris/model"
```

Comment:
I know this is all WIP, so I'm going to pretend to not be bugged by commented out imports

Reply:
Don't worry, it's basically being burned to the ground soon.